src/components/Attachments/AttachmentView/index.tsx

```diff
@@ -271,6 +272,16 @@ function AttachmentView({
                     <View>
                         <Text style={[styles.notFoundTextHeader]}>{translate('attachmentView.attachmentNotFound')}</Text>
                     </View>
+                    <Button
+                        text={translate('attachmentView.tryAgain')}
+                        icon={Sync}
+                        onPress={() => {
+                            if (isOffline) {
+                                return;
+                            }
+                            setImageError(false);
```

Comment:
I am yet to test but I can see `setImageError` is a boolean state value. I am not sure what's the logic of retrying. Let me pull the code and check locally.